config/initializers/rack_attack.rb

```diff
@@ -12,9 +12,22 @@ def self.protected_path?(request)
     PROTECTED_ROUTES.any? { |route| request.path.start_with?(route) }
   end
 
+  def self.api_request?(request)
+    request.path.starts_with?("/api/")
+  end
+
+  def self.auth_token(request)
+    request.get_header("HTTP_AUTHORIZATION")
```

Comment:
Do we need to fallback to the ip in case someone tries to hit the api without sending a token?

Reply:
Added fallback throttle to limit to 1000 reqs within 5min